src/vstest.console/Processors/TestAdapterLoadingStrategyArgumentProcessor.cs

```diff
@@ -211,13 +207,19 @@ private void ForceIsolation()
             return;
         }
 
+        EqtTrace.Warning(
+            $"{nameof(TestAdapterLoadingStrategyArgumentExecutor)}.{nameof(ForceIsolation)}: InIsolation setting is forced when {nameof(TestAdapterLoadingStrategy.Explicit)} strategy is used." +
+            "Test's will run in isolation."
+        );
         _commandLineOptions.InIsolation = true;
         _runSettingsManager.UpdateRunSettingsNode(InIsolationArgumentExecutor.RunSettingsPath, "true");
     }
 
     private void ValidateTestAdapterPaths(TestAdapterLoadingStrategy strategy)
     {
-        var testAdapterPaths = _commandLineOptions.TestAdapterPath ?? EmptyStringArray;
+#pragma warning disable CA1825 // Avoid zero-length array allocations
```

Comment:
Rule is currently setup to be only `Info` so I'd say we could even skip the pragma (this would help finding diags to fix when/if we increase the severity of the rule). 

Note that if you don't change, it' ok too, we can always lookup for pragmas.